docs/services/xks/operator-guide.md

```diff
@@ -3,6 +3,239 @@ id: operator-guide
+You can find ready to use pipelines under .ci/ in the terraform repo.
+
+#### Update pipelines
+
+Update the variable azureSubscriptionTemplate. You can find the value under Project settings -> Service Connections
+
+<img alt="Service Connections" src={useBaseUrl("img/operator/project_settings.png")} />
+
+In my case sp-sub-project1-xks
+
+```yaml
+name: $(Build.BuildId)
+
+variables:
+  - name: azureSubscriptionTemplate
+    value: "sp-sub-project1-xks-{0}-owner"
+  - name: terraformFolder
+    value: "governance"
+  - name: sourceBranch
+    value: "refs/heads/main"
+```
+
+Also update the project path in name. Also notice the ref, the ref points to witch version of the module that you are using.
+
+```yaml
+resources:
+  repositories:
+    - repository: templates
+      type: git
+      name: project1/azure-devops-templates
+      ref: refs/tags/2021.03.1
+```
+
+#### Add CI/CD piplines
+
+Once again add a pipeline.
+
+Assuming that you named your repository to terraform
+
+Pipelines -> New pipeline -> Azure Repos Git -> terraform -> Existing Azure Pipelines YAML file
+
+Import the pipeline from the following path: .ci/pipeline-governance.yaml
+
+#### Governence stage done
+
+Hopefully after adding the pipeline the pipeline should automatically trigger and the plan and apply stage should go thourgh without a problem.
+
+### confgiure core
+
+Get a CIDR network for your AKS env per env.
+Define in core/variables/env.tfvars
+
+Run the prepare script to create a location for your state file.
+`make prepare ENV=dev DIR=core`
+
+Othen then that you should edit .ci just like you did for governance.
+Add the pipeline and run it.
+
+### configure AKS
+
+Time to install AKS.
+
+Start to update main.tf to point on your organization/project.
+
+```main.tf
+  fluxcd_v2_enabled = true
+  fluxcd_v2_config = {
+    type = "azure_devops"
+    github = {
+      owner = ""
+    }
+    azure_devops = {
+      org  = "organization1"
+      pat  = data.azurerm_key_vault_secret.azure_devops_pat.value
+      proj = "project1"
+    }
+  }
+```
+
+Notice the org and proj
+
+### Create PAT secret
+
+To make it possible for flux to clone repos from azure devops we need to create a Personal Access Token(PAT).
+
+User Settings -> Personal access tokens -> New Token
+
+<img alt="Settings user" src={useBaseUrl("img/operator/settings_user.png")} />
+
+Create a PAT
+
+<img alt="Create PAT" src={useBaseUrl("img/operator/create_pat.png")} />
+
+Copy the generated key, we will need it for the next step.
+
+### Add PAT to Azure Key Vaults
+
+To make it possible for terraform to reach the PAT in a easy and secure way we have chosen to store the PAT in Azure Key Vaults which you need to add manually.
+
+Now you need access to the azure portal.
+
+In azure portal search for "Key vaults" and pick the core one that matches the unique_suffix that you have specified in global.tfvars, in our case 1234.
+
+Key vaults -> core-1234 -> Secrets -> Generate/Import
```

Comment:
cli example as well

Reply:
Please review the cli commands, it works at least :)